### PR TITLE
chore(trezor): Upgrade Trezor Connect to v9

### PR DESCRIPTION
**Pull request by sime**
Quick POC of integrating Trezor Connect v9 beta.

v9 Docs: https://github.com/trezor/trezor-suite/tree/develop/packages/connect

### CODE REVIEW

**Review by Rob-Ferguson — CHANGES_REQUESTED** [review]
image

It looks like we should be using the `@trezor/connect-web` package instead.

**Comment by sime** [comment]
Hi @Rob-Ferguson 

I just pushed a new package.json with @trezor/connect-web.

Can I run this UI that you screenshotted locally?

**Review by Rob-Ferguson — CHANGES_REQUESTED** [review]
I pulled in the latest change and tried again, but I'm still seeing the same `This package is not suitable to work with browser. Use @trezor/connect-web package instead` error.

**Comment by Rob-Ferguson** [comment]
> Can I run this UI that you screenshotted locally?

The screenshot is from running [Caravan](https://github.com/unchained-capital/caravan), our open-source multisig coordinator, locally on my machine. You should also be able to run it locally and (hopefully) recreate the issue yourself by following these steps:

1. I'm assuming you already have `unchained-wallets` cloned locally, so you should just need to run `npm run compile` from that directory to compile the package.
2. Clone the `caravan` repo locally (preferably under the same parent directory as `unchained-wallets`)
3. Update the `package.json` in the caravan directory to point to the this compiled version of `unchained-wallets` (i.e. update the dependencies section to include `"unchained-wallets": "file:../unchained-wallets"`
4. Run `npm i` and `npm start` from within the `caravan` directory.
5. It should spin up a local Caravan instance at https://localhost:3000/caravan in the browser
6. The Caravan test suite specifically (from screenshot) is at https://localhost:3000/caravan#/test (hosted version available [here](https://unchained-capital.github.io/caravan/#/test)), which can also be accessed by clicking the Test Suite option from the dropdown menu at the top right.
7. From there, load a Trezor device with the seed phrase provided on the test suite page, and you can run through each test manually to validate that the device firmware is producing the exact output our platform expects for various signature/address combinations. This is what the Unchained QA team runs to initially validate each new Trezor firmware release.

The issue I run into is after Caravan spins up, I try to detect a hardware device or start the test suite, and it throws the error immediately, rather than letting me proceed.

cc @waldenraines in case the above doesn't make sense.

File: src/trezor.test.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ import {
 } from "./trezor";
 import {ECPair, payments} from "bitcoinjs-lib";
 
-const TrezorConnect = require("trezor-connect").default;
+const TrezorConnect = require("@trezor/connect").default;
```
Comment (waldenraines) [inline src/trezor.test.js:26]: Should these be importing connect web?

**Comment by sime** [comment]
Thanks for the tip @waldenraines. I pushed a change.

Would you mind trying the tests again @Rob-Ferguson, I have looked into running that UI you detailed.

**Review by Rob-Ferguson — APPROVED** [review]
image

I pulled down the latest commit, rebuilt everything, and started Caravan locally again. This time, I was able to run through the entire test suite with a Trezor Model 1 without issue. 🎉 

I noticed when the Trezor connect window would pop up that it did appear to be referencing the new version 9 - i.e., connect.trezor.io/9 instead of connect.trezor.io/8.

**Comment by sime** [comment]
Hi @Rob-Ferguson are you doing anything special for open the window?

I see you have an environment variable `ENV_TREZOR_CONNECT_URL`, are you specifying anything there (just a guess)?

**Comment by Hannsek** [comment]
Hi @Rob-Ferguson,
I just want to let you know that we have released Connect v9. It is now GA. https://www.npmjs.com/package/@trezor/connect/v/9.0.0

**Comment by steveblackmon** [comment]
travis integration seems to be broken at the moment, but i built this PR locally and confirmed it passed tests

**Pull request merged**